### ISSUE TITLE
Ability to cancel a pull operation

### ISSUE THREAD

**Issue by sbwoodside** [rowanj/gitx#78]
Gitx sometimes seems to get stuck pulling a large set of changes from github. It might be a gitx bug or maybe something else, but in any case there's no way to cancel the operation without Force Quit. Support for Cmd-., a Cancel button, etc, would be helpful.

**Comment by rowanj** [comment]
Agreed, and in general I think a lot less operations need to be 'modal' than the ones that are.
